frontend/src/components/LandingPage/Team/Team.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import {
+  Card, CardTitle, CardText, CardGroup, CardBody
+} from 'reactstrap';
+
+const Team = (props) => {
```

Comment:
Can you change this to a functional component please.

Reply:
Updated to a functional component.